src/main/java/graphql/annotations/processor/retrievers/GraphQLFieldRetriever.java

```diff
@@ -175,6 +189,31 @@ private TypeFunction getTypeFunction(Field field, ProcessingElementsContainer co
         return typeFunction;
     }
 
+    private List<GraphQLArgument> getArgumentsForSimpleConnection() {
+        List<GraphQLArgument> args = new ArrayList<>();
```

Comment:
In simple pagination we want to receive only index & number of elements to receive

Reply:
the index is "after" and the number is "first"

Reply:
but there is no need in before and last
and after does not mean "starting index" so its not compatible